test/e2e/bindata/assets/05_deployment.yaml

```diff
@@ -46,6 +46,5 @@ spec:
             - name: OPERATOR_NAME
               value: "descheduler-operator"
             - name: IMAGE
-              value: # set in e2e
+              value: quay.io/openshift/origin-descheduler:4.14
```

Comment:
Does this have any affect?

Reply:
No, I changed this as I was trying applying the folder directly into a cluster to check things running. I will change it back when things are working.

Can we focus on checking why the test pod is not getting evicted?